src/gt4py/next/program_processors/runners/gtfn.py

```diff
@@ -49,7 +47,11 @@ def decorated_program(
         if out is not None:
             args = (*args, out)
         converted_args = [convert_arg(arg) for arg in args]
-        conn_args = extract_connectivity_args(offset_provider, device)
+        try:
```

Comment:
We have a similar `extract_connectivity_args` in dace backend:
https://github.com/GridTools/gt4py/blob/eb0a0c1b322c2a9c32f2beb9e087ee940b4bd19c/src/gt4py/next/program_processors/runners/dace_common/dace_backend.py#L114

If we tweak the gtfn backend this way, we should do the same in dace backend. I can do it, just please remind me.